src/compositemark/errorband.ts

```diff
@@ -82,8 +89,18 @@ export function normalizeErrorBand(
   );
 
   const is2D = spec.encoding.x !== undefined && spec.encoding.y !== undefined;
-  const bandMark = is2D ? 'area' : 'rect';
-  const bordersMark = is2D ? 'line' : 'rule';
+
+  const bandMark: MarkDef = {type: is2D ? 'area' : 'rect'};
+  const bordersMark: MarkDef = {type: is2D ? 'line' : 'rule'};
+
+  if (markDef.interpolate) {
+    if (is2D) {
+      bandMark.interpolate = markDef.interpolate;
```

Comment:
Actually let's add [`tension`](https://vega.github.io/vega-lite/docs/line.html#properties) and apply it too.